src/components/endpoints/Transfer.vue

```diff
@@ -94,6 +138,17 @@ const transfer = async () => {
           </div>
         </div>
       </div>
+      <div v-else class="field">
+        <label class="label">Token id (!: only bytes32, no uint256)</label>
```

Comment:
Instead of the value comment in label I would rather add example in input placeholder.

Reply:
?